### DISCUSSION TITLE
JavaScript meme

### DISCUSSION THREAD

**Discussion by rh5140** [albertkun/21S-ASIAAM-191A#94] [General]
![image](https://user-images.githubusercontent.com/40257341/114945555-6663c280-9dfe-11eb-9a1f-35933c6e6da6.png)

There's also versions for computer science and math:
![image](https://user-images.githubusercontent.com/40257341/114945580-78456580-9dfe-11eb-87e6-b26d79f288e0.png)
![image](https://user-images.githubusercontent.com/40257341/114945631-8dba8f80-9dfe-11eb-8460-78a2fb2e6d3c.png)

**Comment by albertkun** [comment]
Totally true about SLACK and the node.js! HAHA!

Here's the meme I was referring to:
![image](https://user-images.githubusercontent.com/8574425/114946203-9069b480-9dff-11eb-9085-67d517d24257.png)
